contrib_bots/lib/joker.py

```diff
@@ -0,0 +1,52 @@
+# pip install schedule is necessary
+import schedule
+import time
+import json
+import threading
+import os
+from random import choice
+
+cache = os.path.expanduser('lib/Joker/jokes.json')
+
+def job(client):
+    with open(cache) as data_file:
+        joke = list(choice(json.load(data_file)).values())[0]
+    key = dict(
+        type='stream',
+        to='The Stage',
+        subject="Daily Joke",
+        content=joke,
+        )
+    client.send_message(key)
+
+class JokerHandler(object):
+    '''
+    This bot allows you to schedule daily alerts. It
+    looks for messages starting with '@joke'.
+    In this example, we joke everyday
+    in a new stream called "The Stage,".
+    '''
+
+    def usage(self):
+        return '''
+            Users should preface messages with "@joke".
+            Before running this, make sure to create a stream
+            called "The Stage" that your API user can send to.
+            '''
+
+    def triage_message(self, message):
+        # return True if we want to (possibly) response to this message
+        original_content = message['content']
+        # This next line of code is defensive, as we
+        # never want to get into an infinite loop of posting follow
+        # ups for own follow ups.
+        is_link = original_content.startswith('@joke')
```

Comment:
`is_link` is a good name?

should be enough to make: `return original_content.startswith('@joke')`